src/software/simulated_tests/simulated_test_fixture.cpp

```diff
@@ -134,11 +112,20 @@ void SimulatedTestFixture::sleep(
 }
 
 void SimulatedTestFixture::runTest(
+    const Field &field, const BallState &ball,
```

Comment:
Doesn't look to me that the field parameter is ever used

Reply:
fixed